addon/ng2/tasks/format.js

```diff
@@ -0,0 +1,23 @@
+/* jshint node: true */
+'use strict';
+
+var Promise = require('ember-cli/lib/ext/promise');
+var Task = require('ember-cli/lib/models/task');
+var exec = Promise.denodeify(require('child_process').exec);
```

Comment:
Ah yes - guilty as charged. I copy pasted an existing task and took it down to it's bare minimum. The chalk require was there as is, and me not being fully knowledgeable of `ember-cli`, simply left it.